### PR TITLE
Update Converter index.mdx

### PR DESCRIPTION
**Pull request by herrdivad**
fixed Link to images and adding new text to "Specific settings" headline.

### CODE REVIEW

File: docs/chemconverter/index.mdx [hunk 1]
```diff
@@ -105,8 +105,27 @@ Currently there are four used classes:
...
+A list of physical units containing the most common units with and without prefixes.
+If you need additional units, please tell us or use the "scalar operation" feature to convert the column values.
+
+##### Table columns
+
+A drop-down menu for defining the x and y column values while a preview can be seen on the end of the left side as "Input table data".
+Currently only 2D data extraction is possible so if you need more, you can create additional Output tables and merge them together with external tools.
+Alternative you can use the "column operation" feature to add, subtract, divide or multiply columns together.
+
+In a future version it will become possible to create profiles without Output tables if you intend to use the converter only for Metadata extraction like described below.
```
Comment (JanCBrammer) [inline docs/chemconverter/index.mdx:125]: What section does "like described below" refer to? Should there be a link?

Comment (herrdivad) [reply docs/chemconverter/index.mdx:125]: Later there will be a section named sth. like "Metadata extraction" down below.
But it's not written yet.

**Review by herrdivad — COMMENTED** [review]
Why changing the image paths?
As far as I understand the path /img/converter... does not exists. All pics are stored in /static/img/converter. If I use your path Markdown will not show my pics in the preview.
Please correct me if I don't understand it.

File: docs/chemconverter/index.mdx (hunk 1 above)
Comment (harivyasi) [reply docs/chemconverter/index.mdx]: I removed the text that suggested documentation is available. Please add it again once the documentation is included.

**Review by harivyasi — APPROVED** [review]
I have restored image paths so that they start with `/img/...`. The idea here is to avoid relative file paths. All images that go into `static/img` are automatically available at `/img/...`.

Thanks for adding to the documentation.

**Pull request merged**